modules/account/lib/Manager.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Jelix\Authentication\Account;
+
+use jAuthentication;
+use Jelix\Authentication\Core\AuthSession\AuthUser;
+
+class Manager
+{
+    protected static $daoName = 'account~accounts';
+    
+    protected static $daoProfile = 'daotablesqlite';
```

Comment:
the name of the profile should not indicate the type of the database, because we cannot change it here. "accounts" (the name of the module) is better.